app/admin/categories.rb

```diff
@@ -0,0 +1,34 @@
+ActiveAdmin.register Category do
+  menu label: 'Categorías'
+  remove_filter :gift_categorizations
+  filter :name, label: 'Nombre'
+  filter :created_at, label: 'Fecha de creación'
+  filter :updated_at, label: 'Última actualización'
+  permit_params :name
+
+  show do
+    attributes_table do
+      row 'Nombre', &:name
+      row 'Fecha de creación', &:created_at
+      row 'Última actualización', &:updated_at
```

Comment:
If we want to do translations, I suggest that we use i18n. I believe active admin automatically uses it, so we probably just need to add an `es.yml` file with the translations. For more info, check https://guides.rubyonrails.org/i18n.html